testdata/freq.input.txt

```diff
@@ -0,0 +1,11 @@
+apple
```

Comment:
The test case should at least produce double digit frequencies to test the correct prefixing of the single digit frequencies.

Reply:
Yes indeed. Already done on the other branch 👍